src/common/options/mds.yaml.in

```diff
@@ -1587,3 +1587,30 @@ options:
   - mds
   flags:
   - runtime
+- name: mds_log_trim_threshold
+  type: size
+  level: advanced
+  desc: MDS log trim threshold
+  long_desc: The threshold of the number of log segment that can be trimmed.
```

Comment:
number of log segments trimmed per second?

Reply:
How is it _always_ per second? Are you referring to `mds_log_trim_upkeep_interval`?

Reply:
nvm, ; i was thinking the threshold here means a max limit allowed per unit(like per sec or minute)? but its the total limit, got it.